src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class WorkWithFile {
+    private int valueBuy = 0;
+    private int valueSupply = 0;
+    private int result = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String fromFile = fromFileName;
+        String toFile = toFileName;
+
+        readingData(fromFile);
+        printToFile(valueSupply, valueBuy, result, toFile);
+    }
+
+    public void readingData(String fromFileName) {
+        File file = new File(fromFileName);
+
+        try (BufferedReader reader = Files.newBufferedReader(Paths.get(fromFileName))) {
+            String value = reader.readLine();
+            while (value != null) {
+                String[] values = value.split(",");
+                if (values[0].equals("buy")) {
+                    valueBuy += Integer.parseInt(values[1]);
+                }
+                if (values[0].equals("supply")) {
+                    valueSupply += Integer.parseInt(values[1]);
+                }
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            System.out.println("Can't read file");
+        }
+        result = valueSupply - valueBuy;
+    }
 
+    public void printToFile(int valueDaySupply, int valueDayBay, int result, String toFileName) {
```

Comment:
```suggestion
    public void printToFile(String content, String toFileName) {
```
краще в метод print/write передавати сформований вже ріпорт, так якщо в цій самій програмі тобі потрібно буде ще в якомусь місці записувати щось до файлу ти зможеш перевикористати цей самий метод і не потрібно буде писати новий

Reply:
Ok